sqlite3.go

```diff
@@ -1018,6 +1018,17 @@ const (
 	SQLITE_LIMIT_WORKER_THREADS      = C.SQLITE_LIMIT_WORKER_THREADS
 )
 
+// GetFilename returns the absolute path to the file containing
+// the requested schema. When passed an empty string, it will
+// instead use the database's default schema: "main".
+// See: sqlite3_db_filename, https://www.sqlite.org/c3ref/db_filename.html
+func (c *SQLiteConn) GetFilename(schemaName string) string {
+	if schemaName == "" {
```

Comment:
Any particular reason you decided to actually do this? Why not just let an empty string be passed to the underlying function?

Reply:
Yes. Passing an empty string results in a error. Most `sqlite` users are unaware of a sqlite database connection to have multiple named schemas open (since few use attached tables), and consequently don't know that the default is `main`.

Most people will just get confused, so handling the empty string allows a more intuitive, less surprising, user experience.

Reply:
That said, I also considered splitting into two `func`s, something like:

- `GetFilename()` which is a convenience function for
- `GetFilenameFor(schema string)`